src/Microsoft.PowerShell.Commands.Utility/commands/utility/update-list.cs

```diff
@@ -24,7 +23,7 @@ public class UpdateListCommand : PSCmdlet
     {
         /// <summary>
         /// The following is the definition of the input parameter "Add".
-        /// Objects to be add to the list
+        /// Objects to be add to the list.
```

Comment:
Suggest: change 'to be add' to ' to add'

Reply:
Fixed.